Management_System/Management_System/Enums/TaskEstimation.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Management_System.Enums
+{
+    public enum TaskEstimation
```

Comment:
let's use default values for all enums
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/enum